README.md

```diff
@@ -32,6 +32,14 @@ $ python --version
 
 See the [node-gyp](https://github.com/nodejs/node-gyp) documentation for more information.
 
+## CLI
+
+HSD comes with command-line interface tools `hsd-cli` (to interact with the node
+server) and `hsw-cli` (to interact with the wallet server). These applications
+are available in `/bin` (for example the command `./bin/hsd-cli info` returns
```

Comment:
Nit: `/bin` is an absolute path which usually means the root of the filesystem and not the root of the git repo, I think `bin` is more appropriate.

Reply:
Or even `./bin` (which is the relative location of the file w.r.t. README.md).

Reply:
added a `.`! Also specified the npm package name in this line:

> ... or they may be installed globally with `npm i -g hs-client`